webapp/components/survey/CategoryDetails/store/state/state.js

```diff
@@ -12,7 +12,7 @@ const keys = {
   importSummary: 'importSummary',
   inCategoriesPath: 'inCategoriesPath',
   levelItems: 'levelItems',
-  levelActiveItems: 'levelActiveItems',
+  activeItems: 'activeItems',
 }
 
```

Comment:
this file has reached the amount of readability lines. 
refactor it in sub folder. see doc for complex states.
it can be done in a next pull request if you wish so

Reply:
sure, I was already starting to split it